CPAC/pipeline/engine.py

```diff
@@ -281,6 +283,10 @@ def copy_resource(self, resource, new_name):
         except KeyError:
             raise Exception(f"[!] {resource} not in the resource pool.")
 
+    def update_resource(self, resource, new_name):
+        # move over any new pipe_idx's
+        self.rpool[new_name].update(self.rpool[resource])
```

Comment:
I don't see this function called anywhere. Not necessarily a problem if it's for something planned, just want to check that something intended for inclusion didn't get left out of the branch